docs/content/en/docs/pipeline-stages/cleanup.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Cleanup"
+linkTitle: "Cleanup"
+weight: 60
+featureId: cleanup
+---
+
+Skaffold works with [image builders]({{<relref "/docs/pipeline-stages/builders">}}) and [deployers]({{<relref "/docs/pipeline-stages/deployers">}}) 
+that both have side effects on the environment: 
+
+- resources are created in one or more namespaces in a Kubernetes cluster 
+- images might be created on the local Docker daemon
+- images might be pushed to registries
+- application might have arbitrary side effects 
+    
+Skaffold offers functionality to cleanup some of these side effects:
+
+- Kubernetes resource cleanup - `skaffold delete`, and Ctrl+C for `skaffold dev` and `skaffold debug`   
+- Image pruning - for local Docker daemon images only, on Ctrl+C for `skaffold dev` and `skaffold debug` 
+
+For pushed images in registries and application side effects the user has to take care of cleanup. 
+
+## Kubernetes resource cleanup 
+ 
+After you ran `skaffold run` or `skaffold deploy` and deployed your application to a cluster, running `skaffold delete` will remove all the resources you deployed.
+Cleanup is enabled by default, it can be turned off by `--cleanup=false`. 
+
+## Ctrl + C 
+
+In case of `skaffold dev` and `skaffold debug`, pressing `Ctrl+C` (`SIGINT` signal) will kick off the cleanup process which will run a `skaffold delete` essentially.
+In case the skaffold process can't catch the `SIGINT` signal, the running kubernetes resources still can be cleaned up with `skaffold delete`.
```

Comment:
```suggestion
In case the skaffold process can't catch the `SIGINT` signal, the running kubernetes resources can be cleaned up with `skaffold delete`.
```